### PR TITLE
Nav user mobile

### PR DESCRIPTION
**Pull request by SergejsRom**

### CODE REVIEW

File: project/app/Http/Controllers/PostController.php [hunk 1]
```diff
@@ -45,9 +46,19 @@ public function store(Request $request)
...
-    public function show(Post $post)
+    // public function show(int $postId)
+    // {
+
+    //     $post = Post::where('id', $postId)->first();
+        
+    //     return view('front.show', compact('post'));
+    // }
+    
+    public function show($id)
```
Comment (enzofxx) [inline project/app/Http/Controllers/PostController.php]: Would recommend using model binding.

File: project/database/migrations/2022_12_04_170848_create_posts_table.php [hunk 2]
```diff
@@ -17,7 +17,7 @@ public function up()
             $table->id();
             $table->unsignedBigInteger('user_id')->nullable();
             $table->foreign('user_id')->references('id')->on('users');
-            $table->string('name');
+            $table->string('name')->nullable();
```
Comment (enzofxx) [inline project/database/migrations/2022_12_04_170848_create_posts_table.php]: Better create new migration to edit a column.

File: project/app/Http/Controllers/PostController.php [hunk 3]
```diff
@@ -2,6 +2,7 @@
```
Comment (enzofxx) [inline project/app/Http/Controllers/PostController.php:2]: Should this file exist at all?

**Review by enzofxx — CHANGES_REQUESTED** [review]
Better use model binding, but here, if we use livewire, no controllers should exist. After inicial creation of Database and migrations, better add new migration with changes to column, not edit existing migration.

File: project/app/Http/Controllers/PostController.php (hunk 3 above)
Comment (SergejsRom) [reply project/app/Http/Controllers/PostController.php:2]: Possibly not, fixed.

**Pull request closed without merging**